src/index.ts

```diff
@@ -17,8 +17,14 @@ class LoopSDK {
   private onReject: (() => void) | null = null;
   private overlay: HTMLDivElement | null = null;
   private ticketId: string | null = null;
+  public wallet: {
```

Comment:
We will expose this Wallet interface into a type soon. So would rather to have this as an interface with right type instead